storage/src/main/java/tech/pegasys/teku/storage/server/rocksdb/schema/V3Schema.java

```diff
@@ -57,7 +57,6 @@
       RocksDbVariable.create(3, CHECKPOINT_SERIALIZER);
   RocksDbVariable<Checkpoint> FINALIZED_CHECKPOINT =
       RocksDbVariable.create(4, CHECKPOINT_SERIALIZER);
-  RocksDbVariable<BeaconState> LATEST_FINALIZED_STATE = RocksDbVariable.create(5, STATE_SERIALIZER);
```

Comment:
Just confirming - this doesn't break backwards compatibility because it's not a column itself - there will just be a value stored in the DB which we no longer use?

Reply:
Right - manually tested as well to confirm.